### PR TITLE
Validator views tweaks

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/components/BalancesDiff/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import BigNumber from 'bignumber.js'
+import { COLORS } from '../../../styles/theme/colors'
+import Typography from '@mui/material/Typography'
+import { RoundedBalance } from '../RoundedBalance'
+
+type BalancesDiffProps = {
+  a: string | undefined
+  b: string | undefined
```
Comment (lukaw3d) [inline src/app/components/BalancesDiff/index.tsx:10]: I'd rename to before and after

File: src/app/components/Validators/index.tsx [hunk 2]
```diff
@@ -80,14 +87,24 @@ export const Validators: FC<ValidatorsProps> = ({ isLoading, limit, pagination,
...
       },
       {
         align: TableCellAlign.Right,
-        // TODO: provide delegators when it is implemented in the API
-        content: <>-</>,
+        content: (
+          <>
+            {validator.escrow.num_delegators
+              ? validator.escrow.num_delegators.toLocaleString()
+              : t('common.missing')}
```
Comment (lukaw3d) [inline src/app/components/Validators/index.tsx:105]: This prints "missing" if it is 0

File: src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx [hunk 3]
```diff
@@ -23,10 +23,6 @@ function chartData(t: TFunction, validator: Validator | undefined) {
   const otherShares = activeShares - selfDelegationShares
 
   return [
-    {
-      label: t('validator.active'),
-      value: activeShares,
-    },
```
Comment (lukaw3d) [inline src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx:29]: Was this intentional? Math is still `otherShares = activeShares - selfDelegationShares`

File: src/app/pages/ValidatorDetailsPage/index.tsx [hunk 4]
```diff
@@ -135,13 +134,13 @@ export const ValidatorDetailsView: FC<{
               <dd>{formattedTime}</dd>
             </>
           )}
-          {validator.voting_power && (
+          {!!validator.voting_power && (
```
Comment (lukaw3d) [inline src/app/pages/ValidatorDetailsPage/index.tsx:137]: This still hides info if the value is 0 (not undefined)

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx (hunk 3 above)
Comment (buberdds) [reply src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx:29]: it was intentional. It is used to compute otherShares, but showing active shares in chart does not make sense (as it is "entire pie chart"). side note: there is still a discussion about this section.

File: src/app/components/BalancesDiff/index.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+export const BalancesDiff: FC<BalancesDiffProps> = ({ before, after, ticker }) => {
+  const { t } = useTranslation()
+
+  if (!before || !after) {
+    return t('common.missing')
+  }
+
+  const result = new BigNumber(before).minus(new BigNumber(after))
```
Comment (lukaw3d) [inline src/app/components/BalancesDiff/index.tsx:21]: before and after are flipped, probably during rename from a, b

File: src/app/components/Validators/index.tsx [hunk 6]
```diff
@@ -80,14 +94,24 @@ export const Validators: FC<ValidatorsProps> = ({ isLoading, limit, pagination,
       },
       {
         align: TableCellAlign.Right,
-        // TODO: provide change value when it is implemented in the API
-        content: <>-</>,
+        content: (
+          <BalancesDiff
+            before={validator.escrow.active_balance}
+            after={validator.escrow.active_balance_24}
```
Comment (lukaw3d) [inline src/app/components/Validators/index.tsx:100]: Also flipped

Comment (buberdds) [reply src/app/components/Validators/index.tsx:100]: https://github.com/oasisprotocol/explorer/pull/1509

**Pull request merged**